Resources/Prototypes/_CM14/Entities/Structures/Walls/barrier.yml

```
@@ -0,0 +1,40 @@
+- type: entity
+  parent: DeployableBarrier
+  id: CMDeployableBarrier
+  components:
+  - type: Sprite
+    sprite: _CM14/Structures/Walls/barrier.rsi
+    layers:
+    - state: idle
+    - state: locked
+      map: ["enum.LockVisualLayers.Lock"]
+  - type: Appearance
+  - type: AccessReader
+    access: [["CMAccessMarinePrep"]] # code/game/machinery/deployable.dm line 8
+  - type: Lock
+    locked: false
+    lockOnClick: true
+    lockTime: 1
+    unlockTime: 1
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 600
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 300
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CMSheetMetal:
+            min: 5
+            max: 5
```

As far as I can tell in CM13 these are supposed to have 100 health and drop nothing
```suggestion
    - trigger:
        !type:DamageTrigger
        damage: 100
      behaviors:
        - !type:DoActsBehavior
          acts: [ "Destruction" ]
```